sqlx-core/src/pool/mod.rs

```diff
@@ -306,6 +308,18 @@ impl<DB: Database> Pool<DB> {
     }
 }
 
+#[cfg(feature = "any")]
+impl Pool<Any> {
+    /// Returns the dynamic kind of driver used.
+    ///
+    /// This is mostly relevant for the `Any` driver, since the others have a
+    /// statically known kind.
+    #[cfg(feature = "any")]
+    pub fn get_kind(&self) -> AnyKind {
```

Comment:
```suggestion
    pub fn any_kind(&self) -> AnyKind {
```

The `get_` prefix isn't particularly idiomatic to Rust.